### PR TITLE
[COST-2352] raw daily summary deletes

### PR DESCRIPTION
**Pull request by adberglund**
Fixes [COST-2352](https://issues.redhat.com/browse/COST-2352)

Changes proposed in this PR:
* Use direct SQL statements to delete from the daily summary tables on the Trino path

Testing Instructions:
1. Run standard load test customer data flow

### CODE REVIEW

File: koku/masu/database/report_db_accessor_base.py [hunk 1]
```diff
@@ -448,6 +448,29 @@ def delete_line_item_daily_summary_entries_for_date_range(
         msg = f"Deleted {count} records from {table}"
         LOG.info(msg)
 
+    def delete_line_item_daily_summary_entries_for_date_range_raw(
+        self, source_uuid, start_date, end_date, table=None, filters=None
+    ):
+        if table is None:
+            table = self.line_item_daily_summary_table
+        msg = f"Deleting records from {table} from {start_date} to {end_date}"
```
Comment (Red-HAP) [inline koku/masu/database/report_db_accessor_base.py:456]: Shouldn't `{table}` be `{table._meta.db_table}`

File: koku/masu/database/report_db_accessor_base.py [hunk 2]
```diff
@@ -448,6 +448,29 @@ def delete_line_item_daily_summary_entries_for_date_range(
...
+                AND usage_start <= '{end_date}'
+        """
+        if filters:
+            filter_list = []
+            for key, value in filters.items():
+                filter_list.append(f"AND {key} = '{value}'")
+            filter_str = "\n".join(filter_list)
+            sql += filter_str
+
+        self._execute_raw_sql_query(table, sql, start_date, end_date)
```
Comment (Red-HAP) [inline koku/masu/database/report_db_accessor_base.py:472]:
```suggestion
        sql = f"""
            DELETE FROM {table._meta.db_table}
            WHERE source_uuid = %(source_uuid)s::uuid
                AND usage_start >= %(start_date)s::date
                AND usage_start <= %(end_date)s::date
        """
        if filters:
            filter_list = [f"AND {k} = %s" for k in filters]
            sql += "\n".join(filter_list)
        else:
            filters = {}
        filters["source_uuid"] = source_uuid
        filters["start_date"] = source_date
        filters["end_date"] = end_date

        self._execute_raw_sql_query(table, sql, start_date, end_date, bind_params=filters)
```

**Review by Red-HAP — CHANGES_REQUESTED** [review]

File: koku/masu/database/report_db_accessor_base.py [hunk 3]
```diff
@@ -448,6 +448,31 @@ def delete_line_item_daily_summary_entries_for_date_range(
         msg = f"Deleted {count} records from {table}"
         LOG.info(msg)
 
+    def delete_line_item_daily_summary_entries_for_date_range_raw(
+        self, source_uuid, start_date, end_date, table=None, filters=None
+    ):
+        if table is None:
+            table = self.line_item_daily_summary_table
+        msg = f"Deleting records from {table} from {start_date} to {end_date}"
```
Comment (adberglund) [inline koku/masu/database/report_db_accessor_base.py:456]:
```suggestion
        msg = f"Deleting records from {table._meta.db_table} from {start_date} to {end_date}"
```

File: koku/masu/database/report_db_accessor_base.py [hunk 4]
```diff
@@ -448,6 +448,31 @@ def delete_line_item_daily_summary_entries_for_date_range(
...
+                AND usage_start >= %(start_date)s::date
+                AND usage_start <= %(end_date)s::date
+        """
+        if filters:
+            filter_list = [f"AND {k} = %s" for k in filters]
+            sql += "\n".join(filter_list)
+        else:
+            filters = {}
+        filters["source_uuid"] = source_uuid
+        filters["start_date"] = source_date
```
Comment (adberglund) [inline koku/masu/database/report_db_accessor_base.py:471]:
```suggestion
        filters["start_date"] = start_date
```

**Review by Red-HAP — APPROVED** [review]

**Review by Red-HAP — APPROVED** [review]

**Pull request merged**